charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -141,21 +141,85 @@ data:
                 - name: OT_ENABLED
                   value: {{ .Values.application.ray.openTelemetryCollector.enabled | quote }}
 {{- end }}
-            - image: fluent/fluent-bit:1.9.10
-              name: ray-head-logs
+            - image: {{ .Values.application.ray.proxyImage }}
+              securityContext:
+                runAsGroup: 123
+                privileged: true
+                capabilities:
+                  add: ["NET_ADMIN"]
+              imagePullPolicy: Never
+              name: proxy
+              ports:
+              - containerPort: 6379
+                name: gcs
+                protocol: TCP
+              - containerPort: 8265
+                name: dashboard
+                protocol: TCP
+              - containerPort: 10001
+                name: client
+                protocol: TCP
               resources:
                 limits:
-                  cpu: 100m
-                  memory: 128Mi
+                  cpu: {{ .Values.application.ray.cpu }}
+                  memory: {{ .Values.application.ray.memory }}Gi
                 requests:
-                  cpu: 100m
-                  memory: 128Mi
+                  cpu: {{ .Values.application.ray.cpu }}
+                  memory: {{ .Values.application.ray.memory }}Gi
               volumeMounts:
               - mountPath: /tmp/ray
                 name: log-volume
-              - mountPath: /fluent-bit/etc/fluent-bit.conf
-                name: fluentbit-config
-                subPath: fluent-bit.conf
+              - mountPath: /etc/ca/tls
+                name: ca-tls
```

Comment:
I need to take a look more in-depth on this @akihikokuroda but is it possible to refactorize this variables in some way. I have the feeling that we are repeating them in different parts.

Reply:
The init container generate the certificate and the other containers use the certificate.  So the same volumes are mounted by multiple containers.   I might made some extra copy of the mount in the yaml file.  Please let me know.  Thanks!